api/maintainer/update-username.js

```diff
@@ -1,10 +1,25 @@
-const { MSGS: { INTERNAL_SERVER_ERROR } } = require('../../helpers/constants')
+const { MSGS: { INTERNAL_SERVER_ERROR, ALREADY_EXISTS } } = require('../../helpers/constants')
 
 module.exports = async (req, res, ctx) => {
   try {
     const { username } = req.body
     ctx.log.info('updating maintainer username for %s', req.session.userId)
-    await ctx.db.user.updateUsername({ userId: req.session.userId, username })
+    if (!username) {
+      res.status(400)
+      return res.send({ success: false })
+    }
```

Comment:
This should be covered by the schema and will automatically send back the 400 (if `username` is marked as required). As is I think our test coverage dropped a bit since it will be hard (impossible?) to get these 4 lines covered in unit tests (schema rejection happens before handler is invoked).

Reply:
done